From e11d1049ef728876766061b6b9d4f742a38e6257 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Mon, 6 Aug 2018 19:03:29 +0200
Subject: [PATCH 1/1] riscv: allow native compilation

If environment variable CROSS_COMPILE is not set this indicates native
compilation. In this case we should not set an arbitrary value which is
not applicable for 64bit anyway.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 arch/riscv/config.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/riscv/config.mk b/arch/riscv/config.mk
index 219e66683d..f7a2ffbbc6 100644
--- a/arch/riscv/config.mk
+++ b/arch/riscv/config.mk
@@ -10,10 +10,6 @@
 # Rick Chen, Andes Technology Corporation <rick@andestech.com>
 #
 
-ifeq ($(CROSS_COMPILE),)
-CROSS_COMPILE := riscv32-unknown-linux-gnu-
-endif
-
 32bit-emul		:= elf32lriscv
 64bit-emul		:= elf64lriscv
 
-- 
2.18.0

